src/views/SystemProfile/components/ToolsAndSoftware/index.tsx

```diff
@@ -65,7 +65,14 @@ const SystemToolsAndSoftware = ({ system }: SystemProfileSubviewProps) => {
                           )}
                         </>
                       )}
-                      {/* TODO: handle ELA Purchase field */}
+                      {product.elaPurchase === 'Yes' && (
```

Comment:
We should (maybe not in this PR) try to catch stuff like this on the backend and convert it to a `Bool` type, if it's appropriate (or give it our own custom scalar, if it's possible that it can come back with many different values, not just `Yes` and `No)

Nothing to change here, just a mention for future tech debt maybe!

Reply:
Yeah - we have a YesNoMap utility to print out the different values maybe we could do something similar for handling checking/standardizing the values?